circle.yml

```diff
@@ -0,0 +1,55 @@
+machine:
+  environment:
+    ARCH: "i686"
+    BUILDOPTS: "-j3 VERBOSE=1 FORCE_ASSERTIONS=1 LLVM_ASSERTIONS=1"
+    TESTSTORUN: "all"
+    JULIA_CPU_CORES: 2
```

Comment:
So I'm a little confused as to what you actually get in the free plan. I was able to select up to 4x parallelism, and the way Circle does parallelism is by running completely separate containers, each with 2 vCPUs, in parallel, and keeping each step of the build and test in sync between containers. Now, in the settings they also make it sound like you trade parallel containers for concurrent jobs. I'm not clear on the specifics of that. I should throw a bunch of PRs from different branches at it (to avoid auto-canceling) and see what happens.